### PR TITLE
Fix Responsiveness of Social Media in Footer

### PR DESCRIPTION
**Pull request by fidellim**
The social media button at the far right is overlapping at mobile screen sizes.

<details><summary>Click to view "Before" </summary>
<p>

https://user-images.githubusercontent.com/26820513/233783087-bf96df1f-97d6-414d-96de-9ecae4b6b6c0.mp4

</p>
</details> 

<details><summary>Click to view "After"</summary>
<p>

https://user-images.githubusercontent.com/26820513/233783104-f27948d6-f68b-4760-9804-8ffaf9a26910.mp4

</p>
</details>

### CODE REVIEW

File: src/layouts/MainLayout.vue [hunk 1]
```diff
@@ -73,8 +73,9 @@ q-layout(
     router-view
 
   q-footer(style="height: 200px;").ossph-gradient-bg.q-pb-md.text-center.text-white
-    q-toolbar.q-gutter-sm.q-my-lg
-      q-space
+    q-toolbar(
+      class="soc-med-toolbar"
```
Comment (jofftiquez) [inline src/layouts/MainLayout.vue:77]: Hi, same comment as this one 👉 https://github.com/OSSPhilippines/ossph.org/pull/22#discussion_r1174522096

**Review by jofftiquez — CHANGES_REQUESTED** [review]

File: src/layouts/MainLayout.vue (hunk 1 above)
Comment (fidellim) [reply src/layouts/MainLayout.vue]: > Hi, same comment as this one 👉 [#22 (comment)](https://github.com/OSSPhilippines/ossph.org/pull/22#discussion_r1174522096)

Have made the changes sir @jofftiquez. As well as in #22

Comment (jofftiquez) [reply src/layouts/MainLayout.vue]: Hey thanks @fidellim and please, drop the "sir" 😄

**Review by jofftiquez — APPROVED** [review]

File: src/layouts/MainLayout.vue (hunk 1 above)
Comment (fidellim) [reply src/layouts/MainLayout.vue]: hahaha alright! @jofftiquez 🤝

**Pull request merged**